### PR TITLE
Stan lvl three

### PR DESCRIPTION
**Pull request by dameert**

### CODE REVIEW

File: Entity/UploadedAsset.php [hunk 1]
```diff
@@ -261,7 +261,7 @@ public function getType()
     /**
      * Set user
      *
-     * @param string $user
+     * @param User $user
```
Comment (theus77) [inline Entity/UploadedAsset.php]: Nope it's a string, there is non hard link between the ems core data model and a user entity. Always a soft username string.

File: Entity/View.php [hunk 2]
```diff
@@ -58,7 +58,7 @@ class View
     private $icon;
     
     /**
-     * @var string
```
Comment (theus77) [inline Entity/View.php]: Nope it's a string, there is non hard link between the ems core data model and a user entity. Always a soft username string.

File: Entity/UploadedAsset.php (hunk 1 above)
Comment (theus77) [reply Entity/UploadedAsset.php]: we should probably rename this variable to username

Comment (dameert) [reply Entity/UploadedAsset.php]: Ok I Will take care of this. I see Indeed that doctrine mapping is string. I Will change the @var of the property

File: Entity/View.php (hunk 2 above)
Comment (dameert) [reply Entity/View.php]: This code is about the $options array. That is a json_array in doctrine. In php this is an array.
This is proven by it's usage on many places: 
$view->getOptions()['targetField']

**Pull request merged**